ample/modelling/rosetta_model.py

```diff
@@ -433,9 +563,9 @@ def get_version(self):
         else:
             # Version file is absent in 3.5, so we need to use the directory name
             logger.debug('Version file for Rosetta not found - checking to see if its 3.5 or 3.6')
-            if self.rosetta_dir.endswith(os.sep): self.rosetta_dir = self.rosetta_dir[:-1]
+            self.rosetta_dir = self.rosetta_dir.rstrip(os.sep)
             if self.rosetta_dir.endswith("3.5"):
-                version=3.5
+                version = 3.5
             # 3.6 bundles seem to look like: rosetta_2014.30.57114_bundle
             # elif re.search("rosetta_\d{4}\.\d{2}\.\d{5}_bundle",dirname):
             # Ignore as people change the directory names - just check for the presence of the folders:
```

Comment:
should this not `return True`? alternatively why does the previous `if` need to return `False`? seems inconsistent